### DISCUSSION TITLE
IBM Sarama and librdkafka

### DISCUSSION THREAD

**Discussion by lmtyler** [IBM/sarama#2511] [Q&A] [answered]
With the migration complete and reading this [2461](https://github.com/IBM/sarama/issues/2461), it is not clear in my mind if IBM intends to follow in Shopify's direction and also convert to librdkafka.

My hope is that they do not intend to use librdkafka and continue with pure Go.

**Comment by dnwe** [comment, marked as answer]
Correct, there's no intention for sarama to move away from being anything other than a pure-Go kafka client implementation and it's clear that there is still plenty of demand for that from people building their applications in Go

**Reply by lmtyler** [reply]
Prefect! Thanks for clarifying and quick response!

**Comment by lmtyler** [comment]
Prefect! Thanks for clarifying and quick response!
